gensim/models/ldamodel.py

```diff
@@ -498,7 +501,13 @@ def inference(self, chunk, collect_sstats=False):
             # The optimal phi_{dwk} is proportional to expElogthetad_k * expElogbetad_w.
             # phinorm is the normalizer.
             # TODO treat zeros explicitly, instead of adding 1e-100?
-            phinorm = np.dot(expElogthetad, expElogbetad) + 1e-100
+            dtype_to_eps = {
```

Comment:
Better defined at module level, so `__init__` can make use of the allowed keys? Now the same information is in three places, not DRY.